cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -453,19 +453,27 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         },
 
         getRequestData: function() {
+            var sectionIds = $('.outline-section').map(function(){return this.id;}).get()
+            var subsectionIds = $('.outline-subsection').map(function(){return this.id;}).get()
+            var relative_weeks_due = null;
             if (this.getValue() < 19 && this.getValue() > 0 && $('#grading_type').val() !== 'notgraded') {
-                return {
-                    metadata: {
-                        relative_weeks_due: this.getValue()
-                    }
-                };
-            } else {
-                return {
-                    metadata: {
-                        relative_weeks_due: null
-                    }
-                }
+                relative_weeks_due = this.getValue()
             }
```

Comment:
Really digging the way you handled this method. Great thinking here. Clean, concise, utilizing the default value. It's wonderful